### ISSUE TITLE
Enhancement: Porting Useful Checks from flake8-string-format

### ISSUE THREAD

**Issue by DylanYoung** [peterjc/flake8-sfs#4]
See https://pypi.org/project/flake8-string-format/

There are a number of useful checks there about the number of/missing parameters that apply to both format strings and printf style format strings.

Probably an analogous check could be made for f-string as well (assuming no dynamic modification of locals at least).

**Comment by peterjc** [comment]
I'm not sure that makes sense when people can easily install ``flake8-sfs`` and ``flake8-string-format`` together. At least, this is a big expansion on what I set out to cover with my plugin.

Suggesting adding some f-string specific checks in ``flake8-string-format`` does make sense to me.

**Comment by DylanYoung** [comment]
`flake8-string-format` doesn't handle f-strings or % strings

**Comment by DylanYoung** [comment]
You could certainly leave off the new-style format strings though.  My thoughts are that it's better to have everything in one maintained place, but ultimately you're the maintainer so that's your call :)

**Comment by peterjc** [comment]
Fair point. Both are under the MIT license which helps... I wonder what @xZise thinks?

**Comment by xZise** [comment]
I'm a bit confused what this issue is actually about.

Anyway I don't think my plugin helps with f-strings, as the plugin is there to check whether all (but not to many) parameters are defined. I've worked on some support of %-strings, but as this was a few months ago I don't really know how far I have come. Maybe I'll just upload a branch of it as a starting point.

**Comment by peterjc** [comment]
My understanding was Dylan suggesting that ``flake8-string-format`` and ``flake8-sfs`` be merged in some way, since the both check string formatting things. I'm not convinced yet.
